woocommerce/woocommerce-ios
# [IAP] UI for plan benefit details screens

Part of: #9967, follows #9973


## Description


We're adding the ability to purchase the Woo Express Essential Monthly plan, using In-App Purchase.

This updates the UI for the Upgrades screen to be closer to the designs. In particular, it:
- Adds the `WooPlanFeatureBenefitsView`, and makes each feature group row on the Upgrades screen open the relevant benefits view

~However, it's not a complete implementation of the screen. We still need to:~
- ~Tidy up a lot of constants and numbers~
- ~Review the fonts and other design details~
- ~Make it top aligned when the content is smaller than the screen, not vertically centred~

## Testing instructions


Using a free trial Woo Express store

1. Open the app
2. Tap Upgrade Now
3. Wait for the screen to load
4. Tap in to the feature benefit rows
5. Observe that the views matches the screen capture video below

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/2472348/1ee3c028-994a-4128-9b4e-a35cea7d9ce9



---
